cns/NetworkContainerContract.go

```diff
@@ -75,6 +75,7 @@ type CreateNetworkContainerRequest struct {
 	AllowHostToNCCommunication bool
 	AllowNCToHostCommunication bool
 	EndpointPolicies           []NetworkContainerRequestPolicies
+	WaitingForUpdate           bool // True when NC is waiting for NMA to sync versions/rules
```

Comment:
why this need to be part of cns dnc contract?

Reply:
We need to keep track of whether the NC version is in sync with NMA version

Reply:
moved